dev-docs/bidders/adview.md

```diff
@@ -7,23 +7,22 @@ tcfeu_supported: true
 gvl_id: 1022
 usp_supported: true
 coppa_supported: true
+gpp_supported: true
 schain_supported: true
 dchain_supported: false
-userId: 
 media_types: banner, video, native
-safeframes_ok: true

```

Comment:
If you don't support safe frames anymore, please specify `false`

Reply:
I changed safe frames configuration. Thanks.